partly revert https://github.com/gnuradio/gnuradio/commit/7c87800f1d7e85589678f01281b45451085c7482

so that desktop/icons are installed again instead of relying on a script that would work on TRUEPREFIX..

Index: grc/scripts/freedesktop/CMakeLists.txt
--- grc/scripts/freedesktop/CMakeLists.txt.orig
+++ grc/scripts/freedesktop/CMakeLists.txt
@@ -7,6 +7,21 @@
 
 ########################################################################
 
+# Install desktop
+install(FILES gnuradio-grc.desktop DESTINATION share/applications)
+# Install mime
+install(FILES gnuradio-grc.xml DESTINATION share/mime/packages)
+# Install appstream / metainfo file
+install(FILES org.gnuradio.grc.metainfo.xml DESTINATION share/metainfo)
+# Install icons
+install(FILES grc-icon-256.png DESTINATION share/icons/hicolor/256x256/apps RENAME gnuradio-grc.png)
+install(FILES grc-icon-128.png DESTINATION share/icons/hicolor/128x128/apps RENAME gnuradio-grc.png)
+install(FILES grc-icon-64.png DESTINATION share/icons/hicolor/64x64/apps RENAME gnuradio-grc.png)
+install(FILES grc-icon-48.png DESTINATION share/icons/hicolor/48x48/apps RENAME gnuradio-grc.png)
+install(FILES grc-icon-32.png DESTINATION share/icons/hicolor/32x32/apps RENAME gnuradio-grc.png)
+install(FILES grc-icon-24.png DESTINATION share/icons/hicolor/24x24/apps RENAME gnuradio-grc.png)
+install(FILES grc-icon-16.png DESTINATION share/icons/hicolor/16x16/apps RENAME gnuradio-grc.png)
+
 find_program(HAVE_XDG_UTILS xdg-desktop-menu)
 
 if(UNIX AND HAVE_XDG_UTILS AND ENABLE_POSTINSTALL)
